tables/isolates.csv

```diff
@@ -1197,6 +1197,13 @@ hCoV-19/Hong Kong/HKU-344/2021,Omicron,FALSE,7357684,NULL,TRUE
 hCoV-19/Japan/UT-NCGM02/2020,A,FALSE,418809,NULL,TRUE
 hCoV-19/Russia/MOW-Moscow_PMVL-O16/2021,Omicron,FALSE,7263933,NULL,TRUE
 hCoV-19/USA/GA-EHC-2811C/2021,Omicron,FALSE,7171744,NULL,TRUE
-Chen21e-B.1.1.7,B.1.2,FALSE,NULL,NULL,TRUE
+Chen21e-B.1.1.7,Alpha,FALSE,NULL,NULL,TRUE
 WA1+S:614G,NULL,TRUE,NULL,NULL,TRUE
 WA1+S:501Y+614G,NULL,FALSE,NULL,NULL,TRUE
+B.1.2 full genome, B.1.2,FALSE,NULL,NULL,TRUE
+Chen21e-B.1.351,Beta,FALSE,NULL,NULL,TRUE
+Chen21e-B.1.1.28,B.1.1.28,FALSE,1857098,NULL,TRUE
+Chen21e-B.1.429,Epsilon,FALSE,NULL,NULL,TRUE
+Chen21e-B.1.617.1,Iota/S477N w/o E484K,FALSE,NULL,NULL,TRUE
+Chen21e-B.1.526/S477N,Iota/S477N w/o E484K,FALSE,NULL,NULL,TRUE
+Chen21e-B.1.526/E484K,Iota/S477N w/o E484K,FALSE,NULL,NULL,TRUE

```

Comment:
Iota variant and its derives has another issue pending to be solved by us (#103). However, this is what it should look like:

```csv
Chen21e-B.1.526/S477N,Iota/S477N,FALSE,NULL,NULL,TRUE
Chen21e-B.1.526/S484K,Iota/S484K,FALSE,NULL,NULL,TRUE
```

Reply:
Done